unime/src-tauri/tests/tests/get_state.rs

```diff
@@ -39,6 +39,34 @@ async fn test_get_state_create_new() {
     .await;
 }
 
+#[tokio::test]
+#[serial_test::serial]
+async fn test_locale_persistency_during_onboarding() {
+    setup_state_file();
+    setup_stronghold();
+
+    // Deserializing the AppStates and Actions from the accompanying json files.
+    let state1 = json_example::<AppState>("tests/fixtures/states/no_profile_nl-NL_redirect_welcome.json");
```

Comment:
Can you give it a more speaking name instead of `state1`, such as `state_initial` or `state_before`? (Same applies for `state2` which could have a name such as `state_expected` or `state_after`)?

Reply:
this is the standard format we use for every one of our tests. I definitely take your point but i rather stay consistent or change all tests accordingly

Reply:
All the 'integration tests' in the format we're using right now were meant to be temporary since they were supposed to be generated from mermaid diagrams. So naming actions and states didn't really matter since it would have been abstracted away anyway. 

Since continuation of this is postponed to the 'hardening phase', as well as the fact that there are some other considerations to be made about the current 'testing format' imo it is better to be consistent now, and leave all testing framework specific considerations until we've (time to) come up with something better.